core/store/orm/config.go

```diff
@@ -158,9 +158,27 @@ func init() {
 		MinRequiredOutgoingConfirmations: 39, // mainnet * 13
 	}
 
+	// Optimism is an L2 chain. Pending proper L2 support, for now we rely on Optimism's sequencer
+	optimism := ChainSpecificDefaultSet{
+		EthGasBumpThreshold:              0, // Never bump gas on optimism
+		EthFinalityDepth:                 1, // Sequencer offers absolute finality as long as no re-org longer than 20 blocks occurs on main chain, this event would require special handling (new txm)
+		EthHeadTrackerHistoryDepth:       10,
+		EthBalanceMonitorBlockDelay:      0,
+		EthTxResendAfterThreshold:        5 * time.Second,
+		GasUpdaterBlockHistorySize:       0,               // Force an error if someone set GAS_UPDATER_ENABLED=true by accident; we never want to run the gas updater on optimism
+		HeadTimeBudget:                   1 * time.Second, // Actually heads on Optimism happen every time a transaction is sent so it could be much more frequent than this. Will need to observe in practice how rapid they are and maybe implement special casing
```

Comment:
Should we immediately lower this to 1/60 s as we expect ~60 tps?

Reply:
The node can't handle that.

Reply:
Would it make sense to start a little higher, like `0.1 s` for example. That is 1/6 of the expected capacity.